pages/game.js

```diff
@@ -23,22 +31,52 @@ class Game extends React.Component {
     };
   }
 
+  sleep = time => new Promise(resolve => setTimeout(resolve, time));
+  poll = (p, time) => p().then(this.sleep(time).then(() => this.poll(p, time)));
+
+  getClock = async () => {
+    const { game } = this.props;
+    const { publicRuntimeConfig } = getConfig();
+    const url = `${publicRuntimeConfig.BASEURL}`;
+
+    const result = await fetch(
+      `${url}/api/boxscore/${game.basicGameData.gameId}`,
+    );
+    const g = await result.json();
+    return g;
+  };
+
+  componentDidMount() {
+    this.poll(
+      () =>
+        new Promise(() =>
+          this.getClock().then(result => {
+            this.setState({
+              clock: result.basicGameData.clock,
+              period: result.basicGameData.period,
+            });
+          }),
+        ),
+      60000,
+    );
+  }
+
   render() {
     const {
       game,
       game: {
-        basicGameData: { hTeam, vTeam, clock },
+        basicGameData: { hTeam, vTeam },
       },
     } = this.props;
 
-    return (
+    const { clock, period } = this.state;
+    return period ? (
```

Comment:
I think maybe a null check would be cleaner and easier to read
```suggestion
    return ({ !!period && ... })
```

Reply:
i like the null check!